modules/cnf-about-nic-support-irq-affinity-setting.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-low-latency-tuning.adoc
+
+:_content-type: CONCEPT
+[id="about_nic_support_irq_affinity_setting_{context}"]
+= About NIC support of IRQ affinity setting
+
+Some IRQ controllers lack support for IRQ affinity setting and will always expose all online CPUs as the IRQ mask.
```

Comment:
I would put at the end of this line the sentence "These IRQ controllers effectively run on CPU 0", removing it from its current location. That sentence is related to this one.